package-lock.json

```diff
@@ -1444,6 +1444,21 @@
         }
       }
     },
+    "@babel/runtime": {
```

Comment:
Is this needed by the new additions of `i18next` or `jquery-i18next`? 

Did not expect to see anything with `babel` changing in this PR.

Reply:
I suppose so? It just happened from using `npm install`. There's already a ton of other `babel` dependencies in there.